package.json

```diff
@@ -72,7 +72,7 @@
     "electron": "^11.2.1",
     "electron-builder": "^22.9.1",
     "electron-builder-notarize": "^1.2.0",
-    "electron-mock-ipc": "^0.3.9",
+    "electron-mock-ipc": "brimdata/electron-mock-ipc#master",
```

Comment:
Are we forking this bc we need unreleased features from upstream?

Reply:
Correct, I opened this issue: https://github.com/h3poteto/electron-mock-ipc/issues/361

Then found a way to fix it here: https://github.com/h3poteto/electron-mock-ipc/pull/363

Once that gets merged and a new release is cut, we can remove the fork..